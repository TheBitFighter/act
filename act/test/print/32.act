defchan mychan <: chan(int,int) (bool a) { }

defproc test()
{
  mychan x;
}

test t;
