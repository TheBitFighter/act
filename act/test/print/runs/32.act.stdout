defproc test ();
defchan mychan <: chan(int<32>,int<32>) (bool a);

defproc test ()
{

/* instances */
mychan x;

/* connections */
}

defchan mychan <: chan(int<32>,int<32>) (bool a)
{

/* instances */

/* connections */
}


/* instances */
test t;

/* connections */
